src/library_egl.js

```diff
@@ -428,7 +435,12 @@ var LibraryEGL = {
         {{{ makeSetValue('value', '0', '0x30A0' /* EGL_OPENGL_ES_API */, 'i32') }}};
         return 1;
       case 0x3098: // EGL_CONTEXT_CLIENT_VERSION
-        {{{ makeSetValue('value', '0', '2' /* GLES2 context */, 'i32') }}}; // We always report the context to be a GLES2 context (and not a GLES1 context)
+	if (typeof WebGL2RenderingContext !== 'undefined'
```

Comment:
Doing a `instanceof` check feels a bit brittle cross-browser wise. Let's instead look up the `version` field from the context attributes, see https://github.com/kripken/emscripten/blob/master/src/library_gl.js#L535